### ISSUE TITLE
CanESM5-tasmax.yaml fails cleaning validations for ssp585, ssp126

### ISSUE THREAD

**Issue by brews** [ClimateImpactLab/downscaleCMIP6#345]
CanESM5-tasmax.yaml fails cleaning validations for ssp585, ssp126 in https://argo.cildc6.org/workflows/default/e2e-tasmax-jobs-d4tst

Blocking progress on #189 

Log from failing step in ssp126:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CCCma/CanESM5/ssp126/r1i1p1f1/day/tasmax/gn/v20190429.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 108405
```

For ssp585:
```
Validating gs://clean-b1dbca25/cmip6/ScenarioMIP/CCCma/CanESM5/ssp585/r1i1p1f1/day/tasmax/gn/v20190429.zarr
Traceback (most recent call last):
  File "/argo/staging/script", line 18, in <module>
    _test_timesteps(ds, data_type, time_period)
  File "/opt/dodola/dodola/core.py", line 645, in _test_timesteps
    assert (
AssertionError: projection cmip6 file is missing timesteps, only has 64605
```

**Comment by brews** [comment]
Tagging in @dgergel!

**Comment by dgergel** [comment]
@brews the same thing as in ACCESS-ESM1-5 is happening here for ssp126, the input zarr extend to 2300.  For ssp585, it extends to 2180. 

I've created an issue in dodola to update the validation code to have a more helpful assertion error: https://github.com/ClimateImpactLab/dodola/issues/139

**Comment by brews** [comment]
Closed with PR #365
